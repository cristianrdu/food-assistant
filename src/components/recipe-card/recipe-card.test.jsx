import { shallow } from 'enzyme';
import React from 'react';
import  RecipeCard from './recipe-card';

it('expects to render the Recipe Card component', () => {
    const mockMatch = '/';
    const mockData = {id:'test',recipe: {img:'alt', recipeName: 'test', mealType: 'dinner'}, searchKeyword: undefined}
    expect(shallow(<RecipeCard match = {mockMatch} recipeData = { mockData }/>)).toMatchSnapshot()
})