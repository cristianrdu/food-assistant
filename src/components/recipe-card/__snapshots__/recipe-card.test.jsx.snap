// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expects to render the Recipe Card component 1`] = `ShallowWrapper {}`;
